templates/pages/environment.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "Environment",
+  "layout": [
+    {
+      "type": "widgets",
+      "content": [
+        {
+          "name": "Deployment Button",
+          "x": 0,
+          "y": 0,
+          "width": 3,
+          "height": 3,
+          "definition": "deploymentButton",
+          "configuration": {
+          },
+          "drillDownPages": {
+          }
+        },
+        {
+          "name": "Deployments view",
+          "width": 12,
+          "height": 28,
+          "definition": "deploymentsView",
+          "configuration": {
+            "customPollingTime": 10,
+            "mapOpenByDefault": false,
+            "mapHeight": 300,
+            "fieldsToShow": [
+              "status",
+              "name",
+              "blueprintName",
+              "location",
+              "subenvironmentsCount",
+              "subservicesCount"
+            ],
+            "pageSize": 50,
+            "sortColumn": "created_at",
+            "sortAscending": null,
+            "filterId": "csys-service-filter",
```

We probably want this to be

```suggestion
            "filterId": "csys-environment-filter",
```